docs/releasenotes/1.3_migration_guide.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: default
+title: v1.3.0 Migration Guide
+parent: pages.release_notes
+nav_order: 1
+---
+
+# v1.2.0 Migration Guide
+{: .no_toc }
+
+## Table of contents
+{: .no_toc .text-delta }
+
+1. TOC
+{:toc}
+
+---
+
+## Overview
+
+Hyperledger FireFly v1.3.0 is a feature release that includes changes around event streaming, contract listeners, define/publish APIs as well as a range of general fixes.
+
+For the most part, upgrading from v1.2.x to v1.3.0 should be a seamless experience, but there are several important things to note about changes between the two versions, which are described in detail on this page.
+
+## API definition/publication considerations
+
+**As of FireFly v1.3.0, by default, contract interfaces, contracts APIs, and token pools are unpublished when they are created.**
```

Comment:
Two notes that I think should be added here:

1. This only applies to multiparty namespaces
2. This is a new feature, and I think the way we describe it should highlight the increased flexibility that it provides. We have differentiated creating a resource and publishing it, giving users more control over how and when data is shared in a multiparty namespace